nym-wallet/src-tauri/src/operations/mixnet/account.rs

```diff
@@ -127,19 +135,16 @@ async fn _connect_with_mnemonic(
         .map(|(network, urls)| (*network, urls.iter().next().unwrap().clone()))
         .collect();
 
-    // Run connection tests on all nyxd and nym-api endpoints
-    let (nyxd_urls, api_urls) =
-        run_connection_test(untested_nyxd_urls, untested_api_urls, &config).await;
+    let nyxd_urls = pick_good_nyxd_urls(&default_nyxd_urls, &nyxd_urls).await?;
```

Comment:
For symmetry I think it makes sense to extract out the url selection logic for the api urls in the same way (awaut from `create_clients)